cvmfs-lz/Dockerfile-cc7

```diff
@@ -0,0 +1,28 @@
+##
+## hepsw/cvmfs-cms
+## A container where CernVM-FS is up and running
+##
+FROM hepsw/cvmfs-base-cc7
+LABEL maintainer="Luke Kreczko <kreczko@cern.ch>"
+
+USER root
+ENV USER root
+ENV HOME /root
+ENV VO_LZ_SW_DIR       /cvmfs/cms.cern.ch
```

Comment:
shouldn't this be `/cvmfs/lz.opensciencegrid.org` instead ?

Reply:
It should! Thanks for spotting this.